pandas/core/_numba/executor.py

```diff
@@ -8,15 +8,61 @@
 
 if TYPE_CHECKING:
     from pandas._typing import Scalar
+    from typing import Any
 
 import numpy as np
 
 from pandas.compat._optional import import_optional_dependency
 
 
 @functools.cache
+def make_looper(func, result_dtype, nopython, nogil, parallel):
+    if TYPE_CHECKING:
+        import numba
```

Comment:
Is this because typing checks have trouble with `numba.prange`?

Reply:
This is just moved from the original ``generate_shared_aggregator`` function, but probably I guess.